.github/workflows/continuous-integration.yml

```diff
@@ -265,7 +265,7 @@ jobs:
 
     # deploy will only be run when there is a tag available
     if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-    needs: [quality, test, build_docker_image] # only run after all other stages succeeded
```

Comment:
Why is this removed?

Reply:
Oh, I was trying to see why doesn't the CI remove "Run Tests (3.7)" from the lists. Let me add this back at all the places